src/components/Global/BaseDialog.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-dialog v-model="show" persistent max-width="650" :scrollable="scrollable" elevation scrim="surface-1">
+    <v-card :id="id" class="dialog-card" color="surface-1">
+      <v-card-title class="d-flex justify-space-between pt-4">
+        <h1 v-if="title">{{ title }}</h1>
+        <slot name="title" />
+      </v-card-title>
+      <v-card-text>
+        <slot name="body" />
+      </v-card-text>
+      <div class="dialog-actions d-flex justify-end">
+        <slot name="actions" />
+      </div>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+export default {
+  name: 'BaseDialog',
+  emits: ['update:modelValue'],
+  props: {
+    modelValue: {
+      type: Boolean,
+      required: true,
+    },
+    title: {
+      type: String,
+      default: '',
+    },
+    id: {
+      type: String,
+      required: true,
+    },
+    scrollable: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    show: {
+      get() {
+        return this.modelValue;
+      },
+      set() {
+        // do nothing - parent controls whether dialog is open
+      },
+    },
+  },
+}
+</script>
+
+<style scoped>
+.dialog-card {
+  color: rgba(var(--v-theme-surface-2)) !important;
+  opacity: .92;
+  border: 4px solid rgba(var(--v-theme-surface-2));
+  border-radius: 12px !important;
```

Comment:
@seriouslysean so interestingly the style that I'm bumping into for the color here is this:
<img width="203" alt="image" src="https://user-images.githubusercontent.com/6520704/234443941-0dcd5420-e301-4b4d-af42-2f64ed45aa8b.png">


It's injected by vuetify either for cards or for dialogs, I'm not sure atm. It's tricky to override because vuetify is using important and because we don't have an easy high specificity selector since we want this to apply to the whole dialog.

Interestingly we can make it work by changing the theme color name to `on-surface-1` which is the material design name for the color you put on top of your surfaces. So that could do it, but then it's not necessarily flexible for inverting the colors in some areas of the application. I'm still not sure what's a good way to handle it

Reply:
https://github.com/cuttle-cards/cuttle/issues/398